homeassistant/components/voice_assistant/pipeline.py

```diff
@@ -106,14 +244,20 @@ async def recognize_intent(
             )
         )
 
-        conversation_result = await conversation.async_converse(
-            hass=self.hass,
-            text=intent_input,
-            conversation_id=conversation_id,
-            context=self.context,
-            language=self.language,
-            agent_id=self.pipeline.conversation_engine,
-        )
+        try:
+            conversation_result = await conversation.async_converse(
+                hass=self.hass,
+                text=intent_input,
+                conversation_id=conversation_id,
+                context=self.context,
+                language=self.language,
+                agent_id=self.pipeline.conversation_engine,
+            )
+        except Exception as intent_error:
+            raise IntentRecognitionError(
```

Comment:
We should also log it `_LOGGER.exception("Unexpected error during intent recognition")` or else the exception is lost. Or should it be included in the pipeline?

Reply:
Maybe that too, yeah. It will get sent back as a WebSocket "error" message.